packages/vat-data/test/test-durable-classes.js

```diff
@@ -1,8 +1,8 @@
 // Modeled on test-heap-classes.js
 
-import '@agoric/swingset-liveslots/tools/prepare-test-env.js';
+import { test } from './prepare-test-env-ava.js';
 
-import test from 'ava';
+// eslint-disable-next-line import/order
```

Comment:
we shouldn't need to disable lint. this change is a regression.

Reply:
What's your recommended prelude for using ses-ava?

Reply:
Trick question! I don't recommend it 😁   I haven't had any problems using Ava directly,

```js
import '@agoric/swingset-liveslots/tools/prepare-test-env.js';

import test from 'ava';
```

When it is needed (for console?) then,
```js
import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
```
with care taken that it's the first import. It's brittle because most import sorting tools assume modules with exports won't have side-effects. The one first snippet is stable because it knows the first import is there for its side-effects and it won't reorder it.